lib/Test/Stream/Workflow/Task.pm

```diff
@@ -114,11 +114,16 @@ sub run {
 
     my $unit = $self->{+UNIT};
     my $ctx = $unit->context;
+    my $meta = $unit->meta;
+
+    my $skip;
+    $skip = $meta->{skip} if $meta && defined $meta->{skip};
+    $skip ||= $ctx->debug->_skip; # Private accessor for deprecated thing, just until it goes away
```

Comment:
I will clean this up a little. However I want to keep the new skip system, and the deprecated one I intend to remove on separate lines for easier tracking/removal. I am not worried about performance in this file.